### ISSUE TITLE
Liking a post throws an error

### ISSUE THREAD

**Issue by aadilayub** [fraction/oasis#21]
**What version of this package are you using?**
2.9.1

**What operating system, Node.js, and npm version?**
Fedora 31, Node v12.13.1, npm 6.12.1

(I also have patchwork running in the background)

**What happened?**
I clicked "like" on a post, and got redirected to a page with this text:

```
Error: non-error thrown: {"message":"no async:tangle,branch","name":"Error","stack":"Error: no async:tangle,branch\n    at Object.localCall (/tmp/.mount_patchwzywQgP/resources/app.asar/node_modules/muxrpc/local-api.js:29:13)\n    at Object.<anonymous> (/tmp/.mount_patchwzywQgP/resources/app.asar/node_modules/muxrpc/local-api.js:37:22)\n    at Object.request (/tmp/.mount_patchwzywQgP/resources/app.asar/node_modules/muxrpc/stream.js:48:17)\n    at PacketStream._onrequest (/tmp/.mount_patchwzywQgP/resources/app.asar/node_modules/packet-stream/index.js:161:17)\n    at PacketStream.write (/tmp/.mount_patchwzywQgP/resources/app.asar/node_modules/packet-stream/index.js:134:41)\n    at /tmp/.mount_patchwzywQgP/resources/app.asar/node_modules/muxrpc/pull-weird.js:56:15\n    at /tmp/.mount_patchwzywQgP/resources/app.asar/node_modules/pull-stream/sinks/drain.js:24:37\n    at /tmp/.mount_patchwzywQgP/resources/app.asar/node_modules/pull-goodbye/node_modules/pull-stream/throughs/filter.js:17:11\n    at Object.cb (/tmp/.mount_patchwzywQgP/resources/app.asar/node_modules/packet-stream-codec/index.js:111:11)\n    at drain (/tmp/.mount_patchwzywQgP/resources/app.asar/node_modules/pull-reader/index.js:39:14)"}
    at Object.onerror (/home/aadil/.npm-global/lib/node_modules/@fraction/oasis/node_modules/koa/lib/context.js:113:40)
    at onerror (/home/aadil/.npm-global/lib/node_modules/@fraction/oasis/node_modules/koa/lib/application.js:163:32)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
```

**What did you expect to happen?**
The post gets liked, maybe the heart icon animates or turns pink

**Are you willing to submit a pull request to fix this bug?**
no :(

**Comment by christianbundy** [comment]
Hi! This is my bad. I've recently added a dependence on some plugins that Patchwork doesn't support, which means that you can't actually use some features while Patchwork is running. I'm working on fixing this [here](https://github.com/ssbc/ssb-db/pull/275) with client-side plugins, but it's all very experimental.

Should Oasis just spawn its own server 100% of the time, or should it throw an error if it connects to a server that doesn't implement the necessary plugins?

**Comment by aadilayub** [comment]
@aadilayub I'm not sure. I would prefer if it threw an error and used its own sbot, but I'm not sure what behavior other ssb users prefer/expect.

**Comment by christianbundy** [comment]
Resolved this in #83, thanks again!
